### PR TITLE
Enhance acceptance tests and documentation

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: spec/acceptance/init_spec.rb [hunk 1]
```diff
@@ -21,5 +21,15 @@
     describe process('puppet-summary') do
       its(:args) { is_expected.to match %r{\ -host\ 127.0.0.1\ -port\ 4321} }
     end
+    # see https://github.com/skx/puppet-summary/blob/master/API.md
+    it 'works on get calls' do
+      shell('curl -s http://127.0.0.1:4321/?accept=application/json') do |r|
+        expect(r.stdout).to match(%r{null})
+      end
+    end
+    describe curl_command('http://127.0.0.1:4321/?accept=application/json') do
```
Comment (ekohl) [inline spec/acceptance/init_spec.rb:30]: I'd expect this to work too.

```suggestion
    describe curl_command('http://127.0.0.1:4321/', headers: {'Accept' => 'application/json'}) do
```

Both should work according to the API docs.

**Comment by bastelfreak** [comment]
also contains https://github.com/voxpupuli/puppet-puppet_summary/pull/5

File: spec/acceptance/init_spec.rb [hunk 2]
```diff
@@ -21,5 +21,15 @@
     describe process('puppet-summary') do
       its(:args) { is_expected.to match %r{\ -host\ 127.0.0.1\ -port\ 4321} }
     end
+    # see https://github.com/skx/puppet-summary/blob/master/API.md
+    it 'works on get calls' do
+      shell('curl -s http://127.0.0.1:4321/?accept=application/json') do |r|
```
Comment (binford2k) [inline spec/acceptance/init_spec.rb:26]: is curl guaranteed to be on the path?

Comment (bastelfreak) [reply spec/acceptance/init_spec.rb:26]: yep. somewhere deep down in the nodeset the curl system package is installed and beaker puts almost every existing directory into the $PATH it gets from the running container, it has to be there somewhere. We use the same test on other modules. In the past I tried an absolute path to be 100% sure, but of course that's different on every OS :(
tl;dr this is fine

**Review by dhollinger — APPROVED** [review]
LGTM
